core/src/main/java/org/elasticsearch/cluster/routing/allocation/command/AllocateStalePrimaryAllocationCommand.java

```diff
@@ -70,6 +71,11 @@ public String name() {
         return NAME;
     }
 
+    @Override
+    public Optional<String> getMessage() {
+        return Optional.of("Allocated a stale primary for [" + index + "][" + shardId + "] on node [" + node + "]");
```

Comment:
log messages always start in lower case. Also I wonder if we should give this (and AllocateEmptyPrimaryAllocationCommand) a stronger wording, e.g. "force-allocated a stale primary ...", to make it clearer that this action has been explicitly forced by the user, not the system.